caikit/interfaces/runtime/data_model/prediction_job_management.py

```diff
@@ -0,0 +1,49 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Standard
+from datetime import datetime
+from typing import List
+
+# First Party
+from py_to_proto.dataclass_to_proto import Annotated, FieldNumber
+
+# Local
+from ....core.data_model import DataObjectBase, JobStatus, dataobject
+from .package import RUNTIME_PACKAGE
+
+
+@dataobject(RUNTIME_PACKAGE)
+class PredictionJobInfoRequest(DataObjectBase):
+    """DataModel to request information about a PredictionJob"""
+
+    job_id: Annotated[str, FieldNumber(1)]
```

Comment:
That's a good point. I'll make the update. I originally thought of making it more generic but since training is specific predictions should be as well